Dockerfile

```diff
@@ -1 +1,32 @@
+ARG ARCH="amd64"
+ARG OS="linux"
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
+LABEL maintainer="Armedia, LLC"
 
+ARG ARCH="amd64"
+ARG OS="linux"
+ARG VER="0.22.2"
+ARG PKG="alertmanager"
+ARG SRC="${PKG}-${VER}.${OS}-${ARCH}"
+
+RUN curl \
+		-L "https://github.com/prometheus/${PKG}/releases/download/v${VER}/${SRC}.tar.gz" \
```

Comment:
@drivera-armedia  please download this artifact to our artifacts s3 bucket "arkcase-container-artifacts" and add a script like this to your PR that CodeBuild will run prior to building the dockerfile https://github.com/ArkCase/ark_activemq/blob/fabrice-debase/get-artifacts.sh